template_scripts/packages_centos7.list

```diff
@@ -27,14 +27,14 @@ rsync
 tree
 linux-firmware
 haveged
-mate-notification-daemon
 sudo
 xorg-x11-fonts-100dpi
 xorg-x11-fonts-Type1
 dejavu-sans-fonts
 dejavu-sans-mono-fonts
 dejavu-serif-font
 yum-utils
+--exclude=qubes-artwork
```

Comment:
Why exactly this is needed?

Reply:
I've not identified yet what is exactly pulling this dependency while building template at stage 02. In this case, it requests too much of plymouth and fails.